conf/messages.cy

```diff
@@ -731,7 +731,7 @@ checkYourAnswersForAgents.button.continue = Derbyn a gwneud cais am ddyfarniad
 
 
 agentForTraderCheckYourAnswers.title = Gwiriwch eich atebion cyn anfon eich cais
-agentForTraderCheckYourAnswers.heading = Gwirio’ch atebion
+agentForTraderCheckYourAnswers.heading = Gwirio’ch atebion cyn anfon eich cais
```

Comment:
Please match the title key to the new text as well for confirmation you can see 

I'm also wondering if the text we have for other keys like `checkYourAnswers.heading` is now wrong or should be updated too will look further